src/app/pages/global/global.js

```diff
@@ -5,11 +5,13 @@ import { assign } from 'utils'
 
 import { renderDropdown, renderToggle } from 'components/explorable'
 import GlobalComponent from './global-component'
+import graph from './global-initial-state'
```

Comment:
this import is not needed, because the container is connected and you import that in mapStateToProps on line 14